src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -296,15 +380,20 @@ public async Task<IActionResult> UpdateDisputeContactInfoAsync([Required] string
         }
     }
 
-    private async Task<IActionResult> checkUpdateDisputeInput([Required] string guidHash, CancellationToken cancellationToken)
+    private Boolean CheckGuid([Required] string guidHash, out Guid noticeOfDisputeGuid)

```

Comment:
We already have the an extension method that does this [TryDecodeGuid](https://github.com/bcgov/jag-traffic-courts-online/blob/bf0eec839b02c10d58f333a837c5b7899a9b9873/src/backend/TrafficCourts/Common/HashidsExtensions.cs#L32)

It seems no one is using it.

Reply:
will change